docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
       DANBOORU_REDIS_URL: redis://redis
       DANBOORU_ELASTICSEARCH_HOST: elastic
       DANBOORU_MEMCACHED_SERVERS: memcached
+      DANBOORU_IQDBS_SERVER: http://iqdb:4567
+      DANBOORU_IQDB_ENABLED: true
```

Comment:
```
[liam@fedora e621ng]$ docker-compose up
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.e621.environment.DANBOORU_IQDB_ENABLED contains true, which is an invalid type, it should be a string, number, or a null
```

Reply:
Strange, this worked for me like this. I'll just wrap it in quotes

Reply:
I removed the config value completlly instead. Half the places were just checking for the prences of the server url anyways.